### PR TITLE
Fix callback scope issue while tracing in `MySQL` and `MySQL2` integrations

### PR DESCRIPTION
**Pull request by serkan-ozal**

### CODE REVIEW

**Review by erdogan-yesil — APPROVED** [review]

File: src/integrations/MySQL2Integration.ts [hunk 1]
```diff
@@ -79,33 +114,49 @@ class MySQL2Integration implements Integration {
...
+                    } else {
+                        callback = wrappedCallback;
+                    }
+                    if (overrideInnerCallback) {
+                        sql._callback = wrappedCallback;
+                    }
+
+                    return query.call(this, sql, params, callback);
+
+                    /*
```
Comment (suleymanbarman) [inline src/integrations/MySQL2Integration.ts:151]: Should clean up  comment-out section on code base

Comment (serkan-ozal) [reply src/integrations/MySQL2Integration.ts]: Removed

File: src/integrations/MySQLIntegration.ts [hunk 2]
```diff
@@ -90,48 +125,49 @@ class MySQLIntegration implements Integration {
...
-                    const originalCallback = sequence.onResult;
-
-                    const wrappedCallback = (err: any, res: any) => {
+                    const wrappedCallback = (err: any, res: any, fields: any) => {
                         if (err) {
                             span.setErrorTag(err);
                         }
-                        ThundraLogger.debug(`<MySQLIntegration> Closing MySQL span with name ${span.getOperationName()}`);
-                        span.closeWithCallback(me, originalCallback, [err, res]);
+                        ThundraLogger.debug(`<MySQL2Integration> Closing MySQL span with name ${span.getOperationName()}`);
```
Comment (suleymanbarman) [inline src/integrations/MySQLIntegration.ts]: Log convention should be related `MySQLIntegration`, not `MySQLIntegration2`

File: src/integrations/MySQLIntegration.ts [hunk 3]
```diff
@@ -90,48 +125,49 @@ class MySQLIntegration implements Integration {
...
 
-                    return sequence;
+                    return query.call(this, sql, params, callback);
                 } catch (error) {
                     ThundraLogger.error('<MySQLIntegration> Error occurred while tracing MySQL query:', error);
 
                     if (span) {
                         ThundraLogger.debug(
-                            `<MySQLIntegration> Because of error, closing MySQL span with name ${span.getOperationName()}`);
+                            `<MySQL2Integration> Because of error, closing MySQL span with name ${span.getOperationName()}`);
```
Comment (suleymanbarman) [inline src/integrations/MySQLIntegration.ts]: Log convention should be related `MySQLIntegration`, not `MySQLIntegration2`

**Review by suleymanbarman — APPROVED** [review]

**Pull request merged**